README.md

```diff
@@ -104,77 +110,78 @@ Running `npm run build` in the romcal root directory will generate a packaged ve
 
 ## Usage <a name="usage"></a>
 
-> romcal has been re-written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
+> :information_source: romcal was re-written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
```

Comment:
I think "has been" is more accurate than "was", because it's about something that started in the past and is continuing to happen. Here, the code is still in es6 syntax.
When you use "was", it mean that actually it was rewritten on es6, but not anymore.

Reply:
Well, I really thought that `romcal-es6` (and now the `master` branch) contains (contained) the fully reworked/re-written romcal into ES6. Is that not so? Is not romcal written in ES6 anymore?

On `has been` vs `was`: I know that the former is in _present perfect_ tense and the letter is in the _past_ tense. So:
- if romcal is currently **fully* written in ES6, we should use `was`, as the conversion is already done/finished;
- if romcal is currently **partially** written in ES6 and there are still some parts that _needs to be converted_ and that are planned to be rewritten, we should use `has been`.

Anyway, if romcal **is** written in ES6, we might say it more simply by `romcal is written using ES6`.

Please, tell me in what ‘state’ romcal is. From my point of view, these are the possible ‘states’:
1. fully written in ES6;
2. fully written non-ES6 version of JS;
3. partially converted to ES6 and there is plan to finalise the ES6 conversion (so that romcal would be fully written in ES6);
4. partially converted to ES6 and there is plan to finalise the ES6 conversion (so that romcal would be fully written in non-ES6 version of JS).